### PR TITLE
account + customer classes and tests

### PR DESCRIPTION
**Pull request by magicprog-web**

### CODE REVIEW

File: csharp/TheAccountClass/TheAccountClass/Account.cs [hunk 1]
```diff
@@ -25,17 +31,17 @@ public void Credit(double amount)
         }
         public void Credit(double amount, Account account)
```
Comment (ppiaggesi) [inline csharp/TheAccountClass/TheAccountClass/Account.cs]: this method can be removed

File: csharp/TheAccountClass/TheAccountClass/Account.cs [hunk 2]
```diff
@@ -25,17 +31,17 @@ public void Credit(double amount)
...
-            _balance += amount;
-            account._balance -= amount;
+            PaymentService paymentService = new PaymentService();
+            paymentService.SubmitPayment(account, this, amount);
         }
         public void Debit(double amount)
         {
             _balance -= amount;
         }
         public void Debit(double amount, Account account)
```
Comment (ppiaggesi) [inline csharp/TheAccountClass/TheAccountClass/Account.cs]: this method can be removed

File: csharp/TheAccountClass/TheAccountClass.Tests/PaymentService.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            Customer customer2 = new Customer("2", "Jane", "Doe", "07");
+            Account account2 = new Account(customer2);
+            account2.Credit(2222);
+
+            PaymentService paymentService = new PaymentService();
+            paymentService.SubmitPayment(account1, account2, 1000);
+
+            Assert.AreEqual(111, account1.Balance);
+            Assert.AreEqual(3222, account2.Balance);
+        }
```
Comment (ppiaggesi) [inline csharp/TheAccountClass/TheAccountClass.Tests/PaymentService.cs:25]: You may add another test:
_SubmitPayment_AccountFromHasNotEnoughBalance_ShouldThrowPaymentException_

ShouldThrowPaymentException is just an example, you can define whatever design you want starting from this test

Comment (magicprog-web) [reply csharp/TheAccountClass/TheAccountClass.Tests/PaymentService.cs:25]: This great bank allows unlimited overdraft. (Will have a look on exceptions later)

**Pull request merged**